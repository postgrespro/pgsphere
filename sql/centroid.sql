SELECT centroid(ARRAY[
    spoint(40.7128, -74.0060),
    spoint(34.0522, -118.2437),
    spoint(37.7749, -122.4194)
]);

SELECT centroid('{}');
