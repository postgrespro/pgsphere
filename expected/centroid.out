\set ECHO none
                centroid                
----------------------------------------
 (3.04366980631979 , 0.858938068921891)
(1 row)

NOTICE:  array empty
 centroid 
----------
 
(1 row)


